src/sql/src/names.rs

```diff
@@ -1785,6 +1785,9 @@ impl<'a> Fold<Raw, Aug> for NameResolver<'a> {
                     .collect(),
             ),
             ConnectionKafkaBroker(broker) => ConnectionKafkaBroker(self.fold_kafka_broker(broker)),
+            ConnectionAwsPrivatelink(privatelink) => {
+                ConnectionAwsPrivatelink(self.fold_connection_default_aws_privatelink(privatelink))
```

Comment:
This seems like a bit of magic, and I am not sure it's right.

Reply:
Looks reasonable to me!